kml/serdes_gpx.hpp

```diff
@@ -57,14 +58,19 @@ class GpxParser
   LocalizableString m_name;
   LocalizableString m_description;
   PredefinedColor m_predefinedColor;
-  m2::PointD m_org;
+  geometry::PointWithAltitude m_org;
 
   double m_lat;
   double m_lon;
+  int16_t m_altitude;
```

Comment:
Isn't it a double value? What about deep water tracks? Let's use double here.

Reply:
For now Altitude is int_16t - geometry/point_with_altitude.hpp:9, I think I can just use it for m_altitude instead of int16_t? Min value of int_16t is -32768, should be enough even for Mariana Trench in meters :)

Reply:
Use geometry::Altitude alias.

Reply:
Fixed